src/Sulu/Component/DocumentManager/Subscriber/Behavior/Path/AliasFilingSubscriber.php

```diff
@@ -28,13 +28,19 @@ class AliasFilingSubscriber extends AbstractFilingSubscriber
      */
     private $metadataFactory;
 
+    /**
+     * @var \Doctrine\Inflector\Inflector
```

Comment:
same as above, add here `use \Doctrine\Inflector\Inflector;` for this:

```suggestion
     * @var Inflector
```